Sources/epoch_code/gui/scripts/dynamenu/Epoch_dynamicMenu.sqf

```diff
@@ -108,11 +108,21 @@ _checkConfigs = {
 							format ["%2 %1 call Epoch_dynamicMenu;",_arr + [(configName _x)],getText(_x >> "action")]
 						};
 
+						_tooltip = "";
+						_tooltipcode = getText(_x >> "tooltipcode"),;
```

Comment:
@raymix notced you have an extra comma here, I tested and funny enough it doesn't throw an error

and neither does
EP = 0,121232434234324;